docker-entrypoint.sh

```diff
@@ -29,6 +29,11 @@ if [[ "$CREATE_SUPERUSER" = "1" ]]; then
   echo "Admin user created with credentials admin:adminpass (email: admin@example.com)"
 fi
 
+if [[ "$COMPILE_MESSAGES" = "1" ]]; then
```

Comment:
I think entrypoint (starting of the service) is not a good place for compilation. Could this be done when the docker image is built?

Reply:
I opted to add it here to keep the build script cleaner, and because other manage scripts are also run here. The operation is actually very quick so it shouldn't noticeably increase startup time.

Reply:
Ok, could you rebase this? @a-rmz can you also take a look please?

Reply:
@ahie In general it's a good practice to do any compilation and preparation work on docker build phase, so that we catch the problems early as possible. But in our case it doesn't matter that much since building is done within deployment. It would be great if the docker image would be build just once and used in all envs, there it would matter more.

Reply:
I agree that it would make more sense to have it on the build stage. The commands on the entrypoint are there because they require a database connection, but since this is a code-only change, we could keep it on build.

Reply:
Moved to build phase and rebased.